PubSub/src/Topic.php

```diff
@@ -92,7 +92,21 @@ class Topic
      * @param string $projectId The project Id
      * @param string $name The topic name
      * @param bool $encode Whether messages should be base64 encoded.
-     * @param array $info [optional] A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics)
+     * @param array $info {
+     *     A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
+     *
+     *     @type string name The name of the topic.
+     *
+     *     @type array $labels Key value pairs used to organize your resources.
+     *
+     *     @type array $messageStoragePolicy Policy constraining how messages published to the topic may be stored.
```

Comment:
Grab documentation from [proto](https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto#L114-L122).

Reply:
This option is also not available in the discovery doc yet.

Reply:
I'll further test this to verify it's working since it's missing from discovery doc.